src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgery.cs

```diff
@@ -301,6 +295,15 @@ private AntiforgeryContext GetCookieTokens(HttpContext httpContext)
             antiforgeryContext.HaveGeneratedNewCookieToken = true;
             antiforgeryContext.NewCookieToken = newCookieToken;
 
+            if (antiforgeryContext.HaveGeneratedNewCookieToken)
+            {
+                _logger.NewCookieToken();
+            }
+            else
+            {
+                _logger.ReusedCookieToken();
+            }
+
```

Comment:
Remove this addition.

Would potentially logging multiple times per request. Should instead restore the block where `SaveCookieTokenAndHeader()` is called in `GetAndStoreTokens()`. Then duplicate that logic in `SetCookieTokenAndHeader()`. That way, logging can reliably be done once -- when `HaveStoredNewCookieToken` is `false`.

Note as well that this method is called in cases (`GetTokens()`) where the cookie isn't stored at all.